build_daemon/example/example.dart

```diff
@@ -9,10 +9,17 @@ import 'package:build_daemon/client.dart';
 
 void main(List<String> args) async {
   BuildDaemonClient client;
-  var workingDirectory = Directory.current.path;
+  var workingDirectory =
+      Directory.fromUri(Uri(path: Directory.current.path + '/../example')).path;
```

Comment:
I would recommend using the `path` package here, something like `p.normalize(p.join(p.current, '../example'))`. I am also curious where the `/../example` is coming from (cruft?).

Reply:
Done.